bin/zis-plugin-install.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+
+ZIS_PREFIX=${ZIS_PREFIX:-$USER.DEV}
+ZIS_PARMLIB=${ZIS_PARMLIB:-$ZIS_PREFIX.PARMLIB}
+ZIS_LOADLIB=${ZIS_LOADLIB:-$ZIS_PREFIX.LOADLIB}
+
+mktemp1()
+{
+  FILE=/tmp/nwt-$$-$RANDOM
+  while [ -e $FILE ]
+  do
+    FILE=/tmp/nwt-$$-$RANDOM
+  done
+  echo $FILE
+}
+
+add-plugin-to-parmfile()
+{
+  PARMFILE=$1
+
+  LINE_PREFIX="ZWES.PLUGIN.$PLUGINID="
+  LINE="$LINE_PREFIX""$LMODNAME"
+  if egrep -q "^$LINE_PREFIX" $PARMFILE 
+  then
+    if egrep -q "^$LINE\$" $PARMFILE
+    then
+      >&2  echo "Skipping updating $ZIS_PARMLIB: plugin already registered" 
+      return 1
+    else 
+      >&2  echo "Can't update $ZIS_PARMLIB: the plugin is already"\
+                "registered and points to a different load module"
+      exit 255
+    fi
+  fi
+  echo $LINE >> $PARMFILE
+  return 0
+}
+
+handle-failure()
+{
+  >&2  echo "Installing plugin $PLUGINID failed. Make sure no running ZIS instance"\
+            "or any other process (e.g. ISPF Editor) has locked $ZIS_PARMLIB"\
+            "or $ZIS_LOADLIB"
+  exit 255
+}  
+
+add-plugin-to-libs()
+{(
+  TMPFILE=`mktemp1`
+
+  trap "handle-failure" ERR
+
+  cp "//'$ZIS_PARMLIB(ZWESIP00)'" $TMPFILE
+  trap "rm $TMPFILE" EXIT
+
+  chmod go-rwx $TMPFILE
+
+  if add-plugin-to-parmfile $TMPFILE
+  then
+    >&2  echo "Updating $ZIS_PARMLIB(ZWESIP00)..."
+    cp -v $TMPFILE "//'$ZIS_PARMLIB(ZWESIP00)'"
+  fi
+  
+  >&2 echo "Installing plugin $LMODFILE to $ZIS_LOADLIB($LMODNAME) and $ZIS_PARMLIB"
+  cp -v $LMODFILE "//'$ZIS_LOADLIB'" 
```

Comment:
Can we change this to 
```
cp -X -v $LMODFILE "//'$ZIS_LOADLIB'" 
```
See https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.bpxa400/cpexfile.htm. We fixed a similar bug in the zowe install recently.